localprod/README.md

```diff
@@ -0,0 +1,27 @@
+# Local production setup
+
+This folder allows you to use the
+[manage-tool](https://github.com/OpenSlides/openslides-manage-service) to launch
+the local repository state as a production setup.
+The `config.yml` instructs the manage-tool to include build tags for all service
+images.
+
+## Usage
+
+Run
+
+    ./setup.sh
+
+It will download the latest manage-tool (the `openslides`-executable) - if not
+present already - and setup this directory like a production environment using\
+the `config.yml`.
+
```

Comment:
Either in setup.sh or as an extra step the user mus run `./openslides initial-data`